sound/soc/intel/common/soc-acpi-intel-jsl-match.c

```diff
@@ -47,6 +52,14 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_jsl_machines[] = {
 		.quirk_data = &rt1015_spk,
 		.sof_tplg_filename = "sof-jsl-rt5682-rt1015.tplg",
 	},
+	{
+		.id = "10EC5682",
+		.drv_name = "jsl_max98360a_rt5682",
```

Comment:
follow the convention headset codec-amplifier used for other machines?

Reply:
Ack, jsl_max98360a_rt5682 => jsl_rt5682_max98360a